src/common/walFilter/walFilter.c

```diff
@@ -687,6 +651,40 @@ walFilterNew(const PgControl pgControl, const ArchiveGetFile *const archiveInfo)
 
     OBJ_NEW_BEGIN(WalFilterState, .childQty = MEM_CONTEXT_QTY_MAX, .allocQty = MEM_CONTEXT_QTY_MAX)
     {
+        StringId fork = cfgOptionStrId(cfgOptFork);
+
+        if (fork != CFGOPTVAL_FORK_GPDB)
```

Comment:
The fork variable is used only once
```suggestion
        if (cfgOptionStrId(cfgOptFork) != CFGOPTVAL_FORK_GPDB)
```